src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java

```diff
@@ -286,6 +292,13 @@ private JTextField getJTextFieldSearch() {
 					} catch (OHServiceException e1) {
 						lots = null;
 						OHServiceExceptionUtil.showMessages(e1);
+						return;
+					}
+					if (lots.isEmpty()) {
+						LOGGER.error("No lots found for medical {} and quantity {}", med.getDescription(), qty);
+						JOptionPane.showMessageDialog(MovStockMultipleDischarging.this,
+										MessageBundle.getMessage("angal.sql.anunexpectederroroccurredpleasecheckthelogs.msg"));
```

Comment:
Why isn't this just `MessageDialog.error(this, "angal.sql.anunexpectederroroccurredpleasecheckthelogs.msg")`?

Reply:
Thanks!